yarn.lock

```diff
@@ -5494,6 +5494,11 @@ copy-descriptor@^0.1.0:
   resolved "https://registry.yarnpkg.com/copy-descriptor/-/copy-descriptor-0.1.1.tgz#676f6eb3c39997c2ee1ac3a924fd6124748f578d"
```

Comment:
Shouldn't the `clipboard` package be removed from `yarn.lock`?

Reply:
:heavy_check_mark: